apps/yapms/src/lib/styles/global.css

```diff
@@ -4,8 +4,8 @@
 
 html,
 body {
-	width: 100vw;
-	height: 100vh;
+	width: 100%;
+	height: 100%;
```

Comment:
Should use dvw and dvh here, they dynamically adjust based on if the toolbars are shown or not.

https://web.dev/viewport-units/